Makefile

```diff
@@ -32,6 +37,24 @@ install:
 	mkdir -p $(DESTDIR)/usr/sbin
 	cp shutorial-admin.sh $(DESTDIR)/usr/sbin/shutorial-admin
 
+install-arch:
```

Comment:
I think that we should merge those two install targets in the near future, but it can wait after the merge of this PR. Once the sudo thing is sorted out.

Reply:
I don't think that is possible because, on arch, you can't put anything in the sbin, as it's forbidden by the installation process. Also, the chmod rules are necessary on Arch because ownership and rights are not handled the same as on Debian.